packages/medulas-react-components/src/components/Block/index.tsx

```diff
@@ -75,15 +77,26 @@ interface Props {
   readonly bgcolor?: string;
   //Regular react.component props
   readonly className?: string;
+  //Custom props
+  readonly scroll?: boolean;
 }
 
 // TODO fix once the proper BoxProps have been updated
 // See: https://github.com/mui-org/material-ui/blob/next/packages/material-ui/src/Box/Box.d.ts
 const IovBlock = Box as any; // eslint-disable-line @typescript-eslint/no-explicit-any
 
-const Block = ({ children, display = 'block', ...restProps }: Props): JSX.Element => {
+const useStyles = makeStyles((theme: Theme) => ({
+  scroll: {
+    overflowY: 'scroll',
+    overflowX: 'hidden',
+  },
+}));
+
+const Block = ({ children, display = 'block', scroll, className, ...restProps }: Props): JSX.Element => {
```

Comment:
To me, a Block is a layout container where it grows as the content grows. If we need to support scroll I would suggest creating a new component. Call it, BoxScroll or something like that. 😉